src/VideoStreaming/VideoReceiver.cc

```diff
@@ -417,7 +396,7 @@ gboolean VideoReceiver::_onBusMessage(GstBus* bus, GstMessage* msg, gpointer dat
     Q_UNUSED(bus)
     Q_ASSERT(msg != NULL && data != NULL);
     VideoReceiver* pThis = (VideoReceiver*)data;
-    pThis->_onBusMessage(msg);
+    pThis->busMessage(msg);
```

Comment:
Thanks for the clarification! I've done just that, but also added a signal for Error because some action does need to be taken on the VideoReceiver object, after all.